.github/workflows/build_linux_packages.yml

```diff
@@ -16,11 +19,15 @@ on:
         default: ADHOCBUILD
       amdgpu_families:
         type: string
+      expect_failure:
+        type: boolean
 
 jobs:
   build_linux_packages:
     name: Build Linux Packages
     runs-on: azure-linux-scale-rocm
+    # In the case that the amdgpu family is expect_failure, allow errors
```

Comment:
With the current variable naming and setting `continue-on-error` we could drop this comment.